mappings.json

```diff
@@ -28,14 +34,42 @@
         "FILAP": "Q11874605",
         "FIAHV": "Q2625260"
     },
+    "surfaceWaterBodyCategory": {
+        "@meta": "https://www.wikidata.org/wiki/User:AndreCostaWMSE-bot/WFD/surfaceWaterBodyCategory",
+        "RW": "",
+        "LW": "",
+        "TW": "",
+        "CW": "",
+        "TeW": ""
+    },
+    "swSignificantImpactType": {
+        "@meta": "https://www.wikidata.org/wiki/User:AndreCostaWMSE-bot/WFD/swSignificantImpactType",
+        "ACID": "Q10501532",
+        "CHEM": "Q28854976",
+        "HHYC": "Q28855260",
+        "HMOC": "Q28855259",
+        "LITT": "Q1780867",
+        "LOWT": "Q28854973",
+        "MICR": "Q28854968",
+        "NUTR": "Q12203192",
+        "ORGA": "Q28854984",
+        "SALI": "Q2002254",
+        "TEMP": "Q2914309"
+    },
     "descriptions": {
```

Comment:
Thanks for the strings. I broke this out as a separate task (for clarity) [T163934](https://phabricator.wikimedia.org/T163934)